src/build/index.ts

```diff
@@ -183,6 +185,7 @@ export default class StreamsBuild {
      * @param action            The post-build action to take
      */
     public static async runBuildMake(targetInstance: any, filePaths: string[], action: PostBuildAction): Promise<void> {
+        console.log('window', window.activeTextEditor.document)
```

Comment:
Please remove this `console.log` statement (since we do not want this running in production).